### PR TITLE
Telegraf v1.17.0

### PR DESCRIPTION
**Pull request by ssoroka**

### CODE REVIEW

File: library/telegraf [hunk 1]
```diff
@@ -17,9 +10,16 @@ Directory: telegraf/1.15
...
 Architectures: amd64, arm32v7, arm64v8
 Directory: telegraf/1.16
 
-Tags: 1.16-alpine, 1.16.3-alpine, alpine
+Tags: 1.16-alpine, 1.16.3-alpine
 Directory: telegraf/1.16/alpine
+
+Tags: 1.17, 1.17.0, latest
+Architectures: amd64, arm32v7, arm64v8
+Directory: telegraf/1.17
```
Comment (tianon) [inline library/telegraf:22]: Looking at https://github.com/influxdata/influxdata-docker/tree/22528387700ac8e119269db2adf5ad26a3e3efee/telegraf, either this `Directory:` or the `GitCommit:` above doesn't seem to be quite right? :grimacing:

Comment (ssoroka) [reply library/telegraf:22]: fixed

Comment (tianon) [reply library/telegraf:22]: Was that a commit you meant to push here? :innocent: :heart:

Comment (ssoroka) [reply library/telegraf:22]: Updated, thanks for assist :)

**Pull request merged**